index.html

```diff
@@ -199,9 +199,68 @@ <h2>Multi-Post Stories</h2>
             </div>
 
             <div class="story-button">
-              <button class="button button-1" type="submit">See Project</button> 
+              <button type="button" data-modal-target="#modal" class="button button-1">See Project</button>
             </div>
           </div>
+
+          <div class="modal" id="modal">
+
+            <div class="modalBox">
+
+              <div class="modal-header">
+                <div class="title">Multi-Post Stories</div>
+                <div><button type="button" data-close-button class="close-button"><img src="images/Icon.svg" alt="close"></button>
+                </div>
+              </div>
+
+              <div class="modal-picture"><img src="images/SnapshootPortfolio.png" alt="proect picture"></div>
+
+              <div class="modal-body">
+                <p class="mobile-text">Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the
+                  industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and
+                  scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting ever
+                  since the 1500s, when an unknown printer took a galley of type veris lapoa todoe. </p>
+
```

Comment:
Please ensure that the projects section is created dynamically with the information from the projects array (all the HTML of the section is created when the page loads).